Source/Core/Core/HW/WiimoteEmu/Speaker.cpp

```diff
@@ -106,18 +106,17 @@ void Wiimote::SpeakerData(wm_speaker_data* sd)
 		return;
 	}
 
-	// Speaker Pan
-	unsigned int vol = (unsigned int)(m_options->settings[4]->GetValue() * 100);
-
+	// Speaker Pan using Constant Power Pan Law
+	double pan = m_options->settings[4]->GetValue() / 1.27;
+	double pan_prime = 3.14159265358979323846 * (pan + 1) / 4.0;
```

Comment:
I didn't suggest that, because I remember it causing issues in the past (there's also the case that M_PI isn't a standardized define, but that's being pedantic)